experimental/devices/mfrc522/rf522.go

```diff
@@ -0,0 +1,763 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package mfrc522 controls a Mifare RFID card reader.
+// The code is largely ported from Python library : https://github.com/mxgxw/MFRC522-python
+// The datasheet is available at : https://www.nxp.com/docs/en/data-sheet/MFRC522.pdf
+package mfrc522
+
+import (
+	"errors"
+	"fmt"
+	"periph.io/x/periph/conn/gpio"
+	"periph.io/x/periph/conn/gpio/gpioreg"
+	"periph.io/x/periph/conn/spi"
+	"periph.io/x/periph/experimental/devices/mfrc522/commands"
+	"time"
+)
+
+// Dev defines the RFID reader properties, the SPI bus and GPIO pins.
+type Dev struct {
+	ResetPin      gpio.PinIO
```

Comment:
Do no export, otherwise people can mess with it.
resetPin can probably be gpio.PinOut and irqPin a gpio.PinIn.